src/sdp/ZLIBTextCompressor.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+const pako = require('pako');
```

Comment:
Can we follow the ES6 import syntax here as well?

Reply:
From the issue link here: https://github.com/nodeca/pako/issues/97 , I was under the impression that pako does not support ES6. But after reading it more closely realized, they have added ES6 support. 

I will update the code to follow Es6 format.

Reply:
When I tried to move the code to ES6 format, I realized that all the required zlib apis are not supported in ES6 format. And during local testing realized that the compression was not compatible with the server's compression.

I would hold off on moving it to ES6 format yet.

Reply:
Got it, thanks for the explanation 🙏